### DISCUSSION TITLE
Packages listed in /opt/homebrew/bin are not showing up with "brew list" and I can't remove them

### DISCUSSION THREAD

**Discussion by sokulski** [Homebrew/discussions#4342] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 584588afb4ab557bcc0536f2794b4d043c333bb9
Last commit: 67 minutes ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 030a2e496f0a45a71863f022cfcac0e3e438ca50
Core tap last commit: 3 weeks ago
Core tap branch: master
Core tap JSON: 20 Mar 16:55 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.85.0 => /usr/bin/curl
macOS: 13.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  python@3.10

Warning: Some installed formulae are missing dependencies.
You should `brew install` the missing dependencies:
  brew install node

Run `brew missing` for more details.
```

### Description of issue

Hello!

I think I may have borked something with my Homebrew install. I'm running on an M1 Mac and hav e several packages installed in /opt/homebrew/bin that are not appearing when I run "brew list" and I can't seem to figure out how to uninstall or modify them.

**Comment by SMillerDev** [comment]
What are the packages and what does `brew list` say? Otherwise this is just a lot of guessing

**Comment by sokulski** [comment]
brew list shows:
`boost		ca-certificates	innoextract	lz4		mpdecimal	openssl@1.1	pnpm		readline	zstd
brotli		gdbm		libnghttp2	macos-term-size	nginx		pcre		python@3.10	sqlite
c-ares		icu4c		libuv		mirror-displays	nvm		pcre2		python@3.9	xz`

Whereas /opt/homebrew/bin contains ealint, feathers, npm, nix, nodemon, quasar, and those are just the packages I recognize and recall installing.

Looking at my $PATH, it looks like /opt/homebrew/bin is just the default location that anything that installed via wget went. Does that seem plausible?

**Reply by SMillerDev** [reply]
wget just downloads things. If you were in that directory at the time it would download there, yes.

An easy check is seeing what's in `/opt/homebrew/opt`
